Update CODE_OF_CONDUCT.md (#18)
NixOS/.github: CODE_OF_CONDUCT.md

* Gracefully accepting constructive criticism
* Focusing on what is best for the community
* Showing empathy towards other community members

> * Being considerate of other people's time and energy, and not taking up too much space

Reviewer: ```suggestion
* Being considerate of other people's time and energy, and leaving space for others to be heard
```
?
